source/retryable-reads/tests/changeStreams-db.watch-serverErrors.yml

```diff
@@ -147,7 +147,30 @@ tests:
         operations: [*retryable_operation_fails]
         expectations:
              - *retryable_command_started_event
-
-
-
-
+    -
+        description: "db.watch succeeds after ExceededTimeLimit"
+        failPoint:
+            <<: *failCommand_failPoint
+            data: { failCommands: [aggregate], errorCode: 262 }
+        operations: [*retryable_operation]
+        expectations:
+             - *retryable_command_started_event
+             - *retryable_command_started_event
```

Comment:
Why does `db.watch-serverErrors.yml` expect two events, while the analogous tests in `client.watch` and `db.coll.watch` only expect one event?

Reply:
I think the client.watch and db.coll.watch tests aren't right. They are using `retryReads: false` so they stop after the first failed command. Instead they should be testing the retry behavior like this file (db.watch-serverErrors.yml ) does .

Reply:
Ah, I didn't notice that `client.watch` and `db.coll.watch` were using `clientOptions`. I'll leave new comments to suggest that Rathi remove those.